_translator-files/po/de/Client-Troubleshooting.po

```diff
@@ -45,228 +47,230 @@ msgid ""
 "# Troubleshooting\n"
 " {:.no_toc}\n"
 msgstr ""
+"# Fehlerbehebung\n"
+" {:.no_toc}\n"
 
 #. type: Plain text
 #: ../wiki/en/Client-Troubleshooting.md:12
 msgid "<details markdown=\"1\">"
-msgstr ""
+msgstr "<details markdown=\"1\">"
 
 #. type: Plain text
 #: ../wiki/en/Client-Troubleshooting.md:14
 msgid "<summary>Table of contents</summary>"
-msgstr ""
+msgstr "<summary>Inhaltsverzeichnis</summary>"
 
 #. type: Plain text
 #: ../wiki/en/Client-Troubleshooting.md:16
 msgid "TOC"
-msgstr ""
+msgstr "TOC"
 
 #. type: Plain text
 #: ../wiki/en/Client-Troubleshooting.md:17
 msgid "{:toc}"
-msgstr ""
+msgstr "{:toc}"
 
 #. type: Plain text
 #: ../wiki/en/Client-Troubleshooting.md:19
 msgid "</details>"
-msgstr ""
+msgstr "</details>"
 
 #. type: Plain text
 #: ../wiki/en/Client-Troubleshooting.md:21
 msgid "### Don’t hear any sound/Others don’t hear you?"
-msgstr ""
+msgstr "### Hörst du keinen Sound / Andere hören dich nicht?"
 
 #. type: Plain text
 #: ../wiki/en/Client-Troubleshooting.md:23
 msgid "Start with the simple stuff: make sure your instrument/microphone and headphones are connected to the correct sockets. Make sure no other applications like your browser, video conferencing tool etc. is also using your sound card. You should shut those down when using Jamulus. If that all looks OK and the problem persists, it's likely there's an issue with your sound device settings. This will depend on your particular setup (platform, hardware, software and drivers), so it's best to ask on [the forums](https://github.com/jamulussoftware/jamulus/discussions) for guidance."
-msgstr ""
+msgstr "Beginne mit den einfachen Dingen: Vergewissere dich, dass dein Instrument/Mikrofon und deine Kopfhörer an die richtigen Buchsen angeschlossen sind. Vergewissere dich, dass keine anderen Anwendungen wie Browser, Videokonferenz Programme usw. deine Soundkarte ebenfalls benutzen. Du solltest diese Anwendungen abschalten, wenn du Jamulus verwendest. Wenn das alles in Ordnung ist und das Problem weiterhin besteht, liegt wahrscheinlich ein Problem mit den Einstellungen deines Soundgeräts vor. Dies hängt von deiner speziellen Konfiguration ab (Plattform, Hardware, Software und Treiber), daher frage am besten in [den Foren](https://github.com/jamulussoftware/jamulus/discussions) nach."
```

Comment:
Just a stylistic improvement:

angeschlossen sind. Stelle sicher, dass 
... solltest diese Anwendungen schließen, wenn
(but I think it's also ok like that).

not in den Foren, but im Forum. (Probably the english translation is strange too)

Reply:
Corrected